src/main/java/org/folio/edge/oaipmh/utils/OaiPmhOkapiClient.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.edge.oaipmh.utils;
+
+import io.vertx.core.MultiMap;
+import io.vertx.core.Vertx;
+import java.util.concurrent.CompletableFuture;
+import me.escoffier.vertx.completablefuture.VertxCompletableFuture;
+import org.folio.edge.core.utils.OkapiClient;
+
+public class OaiPmhOkapiClient extends OkapiClient {
+
+  public OaiPmhOkapiClient(OkapiClient client) {
+    super(client);
+  }
+
+  protected OaiPmhOkapiClient(Vertx vertx, String okapiURL, String tenant, long timeout) {
+    super(vertx, okapiURL, tenant, timeout);
+  }
+
+  public CompletableFuture<String> call(MultiMap parameters, MultiMap headers) {
```

Comment:
Instead of just having a single `call` method, this is where you'll eventually have several methods, likely one for each backend module endpoint (e.g. GetRecord/Identify/etc.)

Since this is just a stub, it's fine, but this method will likely be removed/refactored at some point.

Reply:
@craigmcnally,
sure. This will be refactored/removed under client logic implementation in https://issues.folio.org/browse/EDGOAIPMH-15.
Thanks.